### PR TITLE
Merge Release/wallet/1.17.0

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Merge Changes from 1.17.0 version updates back into main.

* 1.17.0

* 1.17.0 (new build)

Co-authored-by: Luna Graysen <luna.graysen@valoraapp.com>

### Backwards compatibility

This is backwards compatible

### CODE REVIEW

File: packages/mobile/Gemfile.lock [hunk 1]
```diff
@@ -260,6 +260,7 @@ GEM
 PLATFORMS
   ruby
   universal-darwin-19
+  universal-darwin-20
```
Comment (gnardini) [inline packages/mobile/Gemfile.lock:263]: Is this change needed?

**Review by gnardini — APPROVED** [review]
A bit surprised by the Gemfile.lock change, but doesn't seem too worrisome, LGTM :)

File: packages/mobile/Gemfile.lock (hunk 1 above)
Comment (MuckT) [reply packages/mobile/Gemfile.lock:263]: This was in the previous release, not sure why it was needed.
